FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Eugene Strelnikov on 01.05.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+final class ManagedCache: NSManagedObject {
+	@NSManaged var timestamp: Date
+	@NSManaged var feed: NSOrderedSet
+}
+
+extension ManagedCache {
+	var localFeed: [LocalFeedImage] {
+		return feed
+			.compactMap { $0 as? ManagedFeedImage }
+			.map { $0.local }
+	}
+
+	static func managedFeedSet(from localFeed: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
```

Comment:
This method could be called `feed` since the context is pretty clear when using it. No need to mention `set` of `managed`
```suggestion
	static func feed(from localFeed: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
```